app/components/Explorer/Witnesses.jsx

```diff
@@ -294,8 +294,8 @@ class WitnessList extends React.Component {
                         ? -1
                         : a.lastConfirmedBlock.timestamp <
                           b.lastConfirmedBlock.timestamp
-                            ? 1
-                            : 0;
+                        ? 1
```

Comment:
There is something in your prettier hook that creates those reformats, or maybe your IDE?